app/assets/stylesheets/index.scss

```diff
@@ -1,57 +1,74 @@
 @import "foundation_and_overrides";
 
 $font-color: #088da5;
-$bg-color: #fff;
+$bg-color: #ECECEA;
+$black: #000000;
 
 body {
-  background-color: $bg-color;
+  background-image: url(/assets/new_year_background.png);
+  background-repeat:repeat;
+  text-align: center;
```

Comment:
Properties should be ordered background-image, background-repeat, font-color, text-align